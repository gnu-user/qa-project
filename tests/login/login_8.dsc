TITLE
    Subsequent Logins
DESCRIPTION
    Attempt to login successfully, and then attempt to run the login 
    transaction again. Verify that the subsequent login attempt fails and 
    produces an error. login_2 must have passed prior to this test, as the 
    login transaction must successfully login to the system before this can be 
    tested.
DEPENDENCIES
    login_2
