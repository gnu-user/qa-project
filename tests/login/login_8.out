Welcome.
> 
Enter username: 
Username accepted.
Available tickets file read successfully.
> 
Invalid transaction, you are already logged in.